Sources/ProjectDescription/TestAction.swift

```diff
@@ -64,6 +70,7 @@ public struct TestAction: Equatable, Codable {
     public static func targets(_ targets: [TestableTarget],
                                arguments: Arguments? = nil,
                                configuration: ConfigurationName = .debug,
+                               attachDebugger: Bool = true,
```

Comment:
Because by default we don't set the value, the default here should be `false`. Otherwise it'll be a breaking change for users.

Reply:
Maybe I am missing something, but before my changes a debugger was always attached implicitly by not setting `selectedDebuggerIdentifier` [here](https://github.com/tuist/tuist/pull/3813/files#diff-1efcecec8991d63da041bac8df08171945a6a432a24add71615c13679e9ee836R373) and hence using its default value, which is `XCScheme.defaultDebugger`. So to prevent this from breaking anything, the default here needs to be `true`, does it not?

Reply:
You're right, to ensure it isn't a breaking change the default would need to be `true`